### DISCUSSION TITLE
Não consigo instalar o Atom pois o AppData é somente leitura fixo

### DISCUSSION THREAD

**Discussion by Adriano-Web** [frontendbr/forum#1266] [Perguntas e Respostas]
Eu não estou conseguindo instalar o Atom editor, o instalador chega na metade do processo de instalação e fecha, logo após me abre uma caixa de diálogo e fornece um _log_ do ocorrido.

Eu chequei o _log_ e está com erro de 'Acesso negado' quando o processo tenta extrair arquivos pra pasta AppData. Qualquer coisa, o _log_ é mostrado [**Ver log**](https://pastebin.com/9wWAgLk9)

Eu sei que isso indica ser necessário a opção somente leitura removida da pasta **AppData\Local** e de todas as outras também, porém, quando eu removo e aplico as alterações, a opção remarca a caixa somente leitura automaticamente.

**Comment by Sergioamjr** [comment]
Tenta abrir o instalador como administrador

**Comment by Adriano-Web** [comment]
> 
> 
> Tenta abrir o instalador como administrador

Ele apresenta o mesmo erro com privilégios administrativos. Eu creio que sejam as permissões. nem no fórum da [atom.io](atom.io) obtive solução.

**Comment by felipefialho** [comment]
@Adriano-Web resolveu?

**Comment by Adriano-Web** [comment]
@LFeh Ainda não.

**Comment by dtfialho** [comment]
@Adriano-Web geralmente quando eu tenho esse problema em windows <s>vulgo ruimdows, uso no trabalho :/</s>, eu já coloco permissão pra tudo em todos os usuários pra não acontecer mais. Faz assim, clica com o **botão direito do mouse > propriedades > segurança > editar** (aqui eu já marco pra permitir controle total pra tudo que tiver pra não ter que setar de novo). Acho que não é a melhor solução mas espero que resolva
:)
